### PR TITLE
Migrate to ros system packages

### PR DESCRIPTION
**Pull request by adityapande-1995**
This PR aims to migrate drake_ros, drake_ros_examples, ros2_examples_bazel_installed to use humble debs instead of the nightly cyclonedds tarball.

- - -
This change is [Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)

### CODE REVIEW

**Review by sloretz — CHANGES_REQUESTED** [review]
> Reviewed 3 of 11 files at r1, 8 of 9 files at r3, 1 of 1 files at r4, 2 of 4 files at r5, 2 of 3 files at r6, 1 of 1 files at r7, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)* status: all files reviewed, 5 unresolved discussions (waiting on @adityapande-1995, @calderpg-tri, and @IanTheEngineer)

___
*[`drake_ros/package.xml` line 30 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9oC9q4qS97ZoTpyKi:-NW9oC9q4qS97ZoTpyKj:b-vl18ek) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/package.xml#L30)):*
> ```XML
>   <depend>visualization_msgs</depend>
>   <depend>tf2_py</depend>
>   <depend>rmw_cyclonedds_cpp</depend>
> ```

This is one of those weird cases where we probably don't want to set it in the package.xml since it's not a dependency of this package. I'd recommend installing it manually in CI for the CMake jobs, and having it in the install_prereqs.sh scripts for the bazel jobs.
___
*[`drake_ros/drake_ros/BUILD.bazel` line 97 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9lVxB36DDH6HW-D14:-NW9lVxC96CCMIk26KW1:b-9v1189) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/drake_ros/BUILD.bazel#L97)):*
> ```Starlark
>     srcs = ["test/geometry_conversions_test.py"],
>     main = "test/geometry_conversions_test.py",
>     rmw_implementation = "rmw_cyclonedds_cpp",
> ```

Ah, this is repeated in a lot of places with the current design (not the fault of this PR). Which RMW implementation to use is probably a project-wide decision. Do you see a way to add `rmw_implementation` to the [base ROS 2 repository attributes](https://github.com/RobotLocomotion/drake-ros/blob/cc773ddbf0a59097f934b65500eb0ffa0e509fa5/bazel_ros2_rules/ros2/defs.bzl#L149) such that if specified at the workspace level then one doesn't have to pass `rmw_implementation` into each individual test rule?
___
*[`drake_ros/setup/install_prereqs.sh` line 22 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9mj1SANim3evlYIB9:-NW9mj1SANim3evlYIBA:b-iss1iq) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/setup/install_prereqs.sh#L22)):*
> ```Shell
> rosdep init
> rosdep update
> rosdep install --from-paths . --rosdistro=humble
> ```

`.` is the current working directory. If the user does `cd drake_ros/setup && ./install_prereqs.sh` then this won't find any packages. There's a bash incantation that gives you the directory containing the script, from which you can concatenate `../` to get the right directory.
___
*[`drake_ros/setup/install_prereqs.sh` line 24 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9mLISE7cfJBSdSeii:-NW9mLIT5UBRQ-3T6p6S:bx0dqg5) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/setup/install_prereqs.sh#L24)):*
> ```Shell
> rosdep install --from-paths . --rosdistro=humble
> 
> apt install python3 python3-toposort
> ```

Recommend adding a note that this is a `bazel_ros2_rules` dependency.
___
*[`drake_ros_examples/setup/install_prereqs.sh` line 23 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9n9qa27e1uojPz2_A:-NW9n9qa27e1uojPz2_B:b-ggx0wz) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros_examples/setup/install_prereqs.sh#L23)):*
> ```Shell
> rosdep init
> rosdep update
> rosdep install --from-paths ../drake_ros --rosdistro=humble
> ```

same comment about this being relative to the current working directory

**Review by adityapande-1995 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)* status: 14 of 17 files reviewed, 4 unresolved discussions (waiting on @calderpg-tri, @IanTheEngineer, and @sloretz)

___
*[`drake_ros/setup/install_prereqs.sh` line 22 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9mj1SANim3evlYIB9:-NWAVHqpA7-NIcI_4ApB:bb6un1j) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/setup/install_prereqs.sh#L22)):*
<details><summary><i>Previously, sloretz (Shane Loretz) wrote…</i></summary><blockquote>

`.` is the current working directory. If the user does `cd drake_ros/setup && ./install_prereqs.sh` then this won't find any packages. There's a bash incantation that gives you the directory containing the script, from which you can concatenate `../` to get the right directory.
</blockquote></details>

Done
___
*[`drake_ros/setup/install_prereqs.sh` line 24 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9mLISE7cfJBSdSeii:-NWAVJBx9lrqthkmYYUX:bb6un1j) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/setup/install_prereqs.sh#L24)):*
<details><summary><i>Previously, sloretz (Shane Loretz) wrote…</i></summary><blockquote>

Recommend adding a note that this is a `bazel_ros2_rules` dependency.
</blockquote></details>

Done
___
*[`drake_ros_examples/setup/install_prereqs.sh` line 23 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9n9qa27e1uojPz2_A:-NWAVMZp4OxLIIlGksmF:bb6un1j) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros_examples/setup/install_prereqs.sh#L23)):*
<details><summary><i>Previously, sloretz (Shane Loretz) wrote…</i></summary><blockquote>

same comment about this being relative to the current working directory
</blockquote></details>

Done
___
*[`drake_ros/package.xml` line 30 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9oC9q4qS97ZoTpyKi:-NWAVPB-6kPi3KwdQM7B:bb6un1j) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/package.xml#L30)):*
<details><summary><i>Previously, sloretz (Shane Loretz) wrote…</i></summary><blockquote>

This is one of those weird cases where we probably don't want to set it in the package.xml since it's not a dependency of this package. I'd recommend installing it manually in CI for the CMake jobs, and having it in the install_prereqs.sh scripts for the bazel jobs.
</blockquote></details>

Done

**Comment by adityapande-1995** [comment]
*[`drake_ros/drake_ros/BUILD.bazel` line 97 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9lVxB36DDH6HW-D14:-NWIsrye4Q-IHS40VCmP:b4d6xqu) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/drake_ros/BUILD.bazel#L97)):*
<details><summary><i>Previously, sloretz (Shane Loretz) wrote…</i></summary><blockquote>

Ah, this is repeated in a lot of places with the current design (not the fault of this PR). Which RMW implementation to use is probably a project-wide decision. Do you see a way to add `rmw_implementation` to the [base ROS 2 repository attributes](https://github.com/RobotLocomotion/drake-ros/blob/cc773ddbf0a59097f934b65500eb0ffa0e509fa5/bazel_ros2_rules/ros2/defs.bzl#L149) such that if specified at the workspace level then one doesn't have to pass `rmw_implementation` into each individual test rule?
</blockquote></details>

The approach I'm trying is to write the default rmw implementation from the workspace file to distro.bzl, and then let ros_py_test and ros_cc_test use this. This might take some testing though due to environment variables being used in multiple places; is it okay if I do this in a separate PR, if this one is urgent ?

**Review by sloretz — CHANGES_REQUESTED** [review]
> Reviewed 3 of 3 files at r8, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)* status: all files reviewed, 2 unresolved discussions (waiting on @adityapande-1995, @calderpg-tri, and @IanTheEngineer)

___
*[`drake_ros/drake_ros/BUILD.bazel` line 97 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9lVxB36DDH6HW-D14:-NWiRjbwEeTf1hWmEPpn:b-9gqttl) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/drake_ros/BUILD.bazel#L97)):*
<details><summary><i>Previously, adityapande-1995 (Aditya Pande) wrote…</i></summary><blockquote>

The approach I'm trying is to write the default rmw implementation from the workspace file to distro.bzl, and then let ros_py_test and ros_cc_test use this. This might take some testing though due to environment variables being used in multiple places; is it okay if I do this in a separate PR, if this one is urgent ?
</blockquote></details>

Sure, a follow up PR is good. I'll open an issue: https://github.com/RobotLocomotion/drake-ros/issues/280

I double checked how that might be implemented, and I think you're right that it should go in the `distro.bzl`.
___
*[`drake_ros/setup/install_prereqs.sh` line 22 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9mj1SANim3evlYIB9:-NWiQnOs-gWgodbbrUCk:b-155kmw) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/setup/install_prereqs.sh#L22)):*
<details><summary><i>Previously, adityapande-1995 (Aditya Pande) wrote…</i></summary><blockquote>

Done
</blockquote></details>

I'm not sure this works, but it's almost there. It can return just a dot:

```
$ dirname -- "$0"
.
```

This trick uses `pwd` to get the full path: https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script
___
*[`drake_ros_examples/setup/install_prereqs.sh` line 23 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9n9qa27e1uojPz2_A:-NWiRAToCOUF7c3u2Kdl:b-ni9t54) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros_examples/setup/install_prereqs.sh#L23)):*
<details><summary><i>Previously, adityapande-1995 (Aditya Pande) wrote…</i></summary><blockquote>

Done
</blockquote></details>

Still an issue: https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script

**Review by adityapande-1995 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)* status: 15 of 17 files reviewed, 2 unresolved discussions (waiting on @calderpg-tri, @IanTheEngineer, and @sloretz)

___
*[`drake_ros/setup/install_prereqs.sh` line 22 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9mj1SANim3evlYIB9:-NWjWJox1vr-MF9JzB3T:bb6un1j) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros/setup/install_prereqs.sh#L22)):*
<details><summary><i>Previously, sloretz (Shane Loretz) wrote…</i></summary><blockquote>

I'm not sure this works, but it's almost there. It can return just a dot:

```
$ dirname -- "$0"
.
```

This trick uses `pwd` to get the full path: https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script
</blockquote></details>

Done
___
*[`drake_ros_examples/setup/install_prereqs.sh` line 23 at r7](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NW9n9qa27e1uojPz2_A:-NWjWLaLD-dN84LT5O5m:bb6un1j) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/4333e95ce56781023b5cdba83b136bd1857412cb/drake_ros_examples/setup/install_prereqs.sh#L23)):*
<details><summary><i>Previously, sloretz (Shane Loretz) wrote…</i></summary><blockquote>

Still an issue: https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script
</blockquote></details>

Done

**Review by EricCousineau-TRI — CHANGES_REQUESTED** [review]
+@EricCousineau-TRI :lgtm: with some nits

> Reviewed 3 of 11 files at r1, 1 of 4 files at r5, 1 of 3 files at r8, 1 of 2 files at r9, 3 of 6 files at r10, 8 of 11 files at r13, 1 of 2 files at r14.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-:-NkHaer22Shc4IeCBSKN:b-hz7umx)* status: 18 of 20 files reviewed, 10 unresolved discussions (waiting on @adityapande-1995, @calderpg-tri, @IanTheEngineer, and @sloretz)

*[a discussion](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHagIB50fN4BP-_5Gj:-NkHagIB50fN4BP-_5Gk:bv0ure5) (no related file):*
Working: Will see if I can address my nits.

___
___
*[`.github/workflows/bazelized_drake_ros.yml` line 37 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHZ8_Z8YMselnJp3VB:-NkHZ8_Z8YMselnJp3VC:bftdi6y) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/.github/workflows/bazelized_drake_ros.yml#L37)):*
> ```YAML
> 
>       # Setup.
>       - name: Free up disk space
> ```

nit This is now redundant w.r.t. above step (see "Free up space").
___
*[`.github/workflows/bazelized_drake_ros.yml` line 77 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkH_lb45f5mdPVjAP-8:-NkH_lb45f5mdPVjAP-9:b-vqohj9) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/.github/workflows/bazelized_drake_ros.yml#L77)):*
> ```YAML
>         working-directory: drake_ros
>       - name: Build drake_ros
>         run: export ROS_DISTRO=humble; bazel build //...
> ```

nit Unclear why this is necessary here and not below.

For now, can we revert to not exporting this env var? (that is how we currently do things in Anzu)
___
*[`drake_ros/setup/install_prereqs.sh` line 2 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkH_wrv0hvVrp8ooSqT:-NkH_wrv0hvVrp8ooSqU:b-ciyfzc) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/drake_ros/setup/install_prereqs.sh#L2)):*
> ```Shell
> #!/bin/bash
> set -uo pipefail
> ```

This should fail on errors, `-e`.
___
*[`drake_ros_examples/setup/install_prereqs.sh` line 2 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHa72VAXwU40kLszPO:-NkHa72VAXwU40kLszPP:b-hxyxei) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/fe146c011407efb19fcc5f3e84d071f0c338cd11/drake_ros_examples/setup/install_prereqs.sh#L2)):*
> ```Shell
> #!/bin/bash
> set -euo pipefail
> ```

nit Same as above. This should fail fast on errors, so we should keep `-e`.
___
*[`drake_ros_examples/setup/install_prereqs.sh` line 30 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHaAQ13LoBk9st8OC_:-NkHaAQ13LoBk9st8OCa:b-l7wqe3) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/drake_ros_examples/setup/install_prereqs.sh#L30)):*
> ```Shell
> 
> # Required for bazel_ros2_rules
> apt install python3 python3-toposort
> ```

nit Needs trailing line.
___
*[`ros2_example_bazel_installed/setup/install_prereqs.sh` line 67 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkH_Al3CQ5zrovMmph7:-NkH_Al3CQ5zrovMmph8:b-epawre) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/ros2_example_bazel_installed/setup/install_prereqs.sh#L67)):*
> ```Shell
>   9276a1e11f03e9f7492f009803c95bddc307993c9ab3c463721c9f6cdaa2ccc1
> 
> if [[ -z "${ROS2_DISTRO_PREFIX:-}" ]]; then
> ```

nit For clarity, consider stating "Install ROS 2 humble if user does not specify a version."
___
*[`ros2_example_bazel_installed/setup/install_prereqs.sh` line 72 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkH_5mm-6h4mcRTE6z_:-NkH_5mm-6h4mcRTE6za:b-d9pceh) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/ros2_example_bazel_installed/setup/install_prereqs.sh#L72)):*
> ```Shell
>   update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
>   export LANG=en_US.UTF-8
>   
> ```

nit Trailing whitespace (here and elsewhere).

**Review by EricCousineau-TRI — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)* status: 18 of 20 files reviewed, 11 unresolved discussions (waiting on @adityapande-1995, @calderpg-tri, @IanTheEngineer, and @sloretz)

___
*[`drake_ros_examples/setup/install_prereqs.sh` line 17 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHbDM04fsS_VhXvNLV:-NkHbDM04fsS_VhXvNLW:b-k01nbs) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/drake_ros_examples/setup/install_prereqs.sh#L17)):*
> ```Shell
> echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" |  tee /etc/apt/sources.list.d/ros2.list > /dev/null
> 
> apt update; apt upgrade
> ```

nit We should avoid unnececssary `apt upgrade`

**Review by EricCousineau-TRI — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)* status: 18 of 20 files reviewed, 12 unresolved discussions (waiting on @adityapande-1995, @calderpg-tri, @IanTheEngineer, and @sloretz)

___
*[`ros2_example_bazel_installed/WORKSPACE` line 84 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHbmuy5o-_0fYd17QZ:-NkHbmuy5o-_0fYd17Q_:b54ikmm) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/ros2_example_bazel_installed/WORKSPACE#L84)):*
> ```Starlark
> ]
> 
> ros2_local_repository(
> ```

nit Having two different targets that resolve to near the same thing is a bit hard to parse.

**Review by EricCousineau-TRI — COMMENTED** [review]
> Reviewed 5 of 5 files at r15, all commit messages.
> Dismissed @sloretz from 2 discussions.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)* status: 19 of 20 files reviewed, all discussions resolved (waiting on @calderpg-tri and @IanTheEngineer)

*[a discussion](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHagIB50fN4BP-_5Gj:-NkHcmy27cYcB6MbTGCC:b-896fix) (no related file):*
<details><summary><i>Previously, EricCousineau-TRI (Eric Cousineau) wrote…</i></summary><blockquote>

Working: Will see if I can address my nits.
</blockquote></details>

Done.

___
___
*[`.github/workflows/bazelized_drake_ros.yml` line 37 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHZ8_Z8YMselnJp3VB:-NkHcd9ZDLPfDfb7-CB6:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/.github/workflows/bazelized_drake_ros.yml#L37)):*
<details><summary><i>Previously, EricCousineau-TRI (Eric Cousineau) wrote…</i></summary><blockquote>

nit This is now redundant w.r.t. above step (see "Free up space").
</blockquote></details>

Done.
___
*[`.github/workflows/bazelized_drake_ros.yml` line 77 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkH_lb45f5mdPVjAP-8:-NkHce1vCC7Rc-WYnsdq:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/.github/workflows/bazelized_drake_ros.yml#L77)):*
<details><summary><i>Previously, EricCousineau-TRI (Eric Cousineau) wrote…</i></summary><blockquote>

nit Unclear why this is necessary here and not below.

For now, can we revert to not exporting this env var? (that is how we currently do things in Anzu)
</blockquote></details>

Done.
___
*[`drake_ros/setup/install_prereqs.sh` line 2 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkH_wrv0hvVrp8ooSqT:-NkHcehyEnfoHOEyBDJM:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/drake_ros/setup/install_prereqs.sh#L2)):*
<details><summary><i>Previously, EricCousineau-TRI (Eric Cousineau) wrote…</i></summary><blockquote>

This should fail on errors, `-e`.
</blockquote></details>

Done.
___
*[`drake_ros_examples/setup/install_prereqs.sh` line 2 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHa72VAXwU40kLszPO:-NkHcgTm2Jky5cDJPxgw:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/fe146c011407efb19fcc5f3e84d071f0c338cd11/drake_ros_examples/setup/install_prereqs.sh#L2)):*
<details><summary><i>Previously, EricCousineau-TRI (Eric Cousineau) wrote…</i></summary><blockquote>

nit Same as above. This should fail fast on errors, so we should keep `-e`.
</blockquote></details>

Done.
___
*[`drake_ros_examples/setup/install_prereqs.sh` line 17 at r14](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278#-NkHbDM04fsS_VhXvNLV:-NkHchIIBDcVTEssFNi6:b-896fix) ([raw file](https://github.com/RobotLocomotion/drake-ros/blob/da9f5c972140c4a4db589d5ade1d6ddf40fa4304/drake_ros_examples/setup/install_prereqs.sh#L17)):*
<details><summary><i>Previously, EricCousineau-TRI (Eric Cousineau) wrote…</i></summary><blockquote>

nit We should avoid unnececssary `apt upgrade`
</blockquote></details>

Done.

**Review by EricCousineau-TRI — APPROVED** [review]
> Reviewed 1 of 2 files at r14, 3 of 3 files at r16, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/RobotLocomotion/drake-ros/278)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @calderpg-tri and @IanTheEngineer)
